src/Directory.Build.props

```diff
@@ -23,6 +23,6 @@
     <VersionPrefix>4.0.0</VersionPrefix>
   </PropertyGroup>
   <PropertyGroup>
-    <TargetFrameworks>net6.0;net5.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net7.0;net6.0;net5.0</TargetFrameworks>
```

Comment:
If you want to drop 5.0 as well, it also needs to be removed from test project. Please let me know @Shazwazza.

Reply:
Can leave for now.